src/Components/test/testassets/Components.TestServer/RazorComponents/Pages/Forms/CustomNumberInput.razor

```diff
@@ -0,0 +1,19 @@
+﻿@using System.Diagnostics.CodeAnalysis;
+@inherits Microsoft.AspNetCore.Components.Forms.InputBase<int>
+
+<input name="@NameAttributeValue" @attributes="AdditionalAttributes" type="text" class="@CssClass" @bind="CurrentValueAsString" />
+
+@code {
+    protected override bool TryParseValueFromString(string value, [MaybeNullWhen(false)] out int result, [NotNullWhen(false)] out string validationErrorMessage)
+    {
+        if(int.TryParse(value, out result))
+        {
+            validationErrorMessage = null;
+            return true;
+        }else
+        {
+            validationErrorMessage = "";
```

Comment:
This is a bit confusing. What scenario does it represent? I'm unsure why someone would return an empty string as a validation message. Is the test specifically about checking for an empty string for some reason?